docs/guides/es2k/running-infrap4d.md

```diff
@@ -0,0 +1,281 @@
+   - [2.2 Setup the Environment](#22-setup-the-environment)
+   - [2.3 Install TLS Certificates](#23-install-tls-certificates)
+   - [2.4 Generate Forwarding Pipeline Binary](#24-generate-forwarding-pipeline-binary)
+      - [2.4.1 Prepare configuration files](#241-prepare-configuration-files)
+      - [2.4.2 Generate binary](#242-generate-binary)
+   - [2.5 Start Infrap4d](#25-start-infrap4d)
+   - [2.6 Set the Pipeline](#26-set-the-pipeline)
+   - [2.7 Configure Forwarding Rule to Receive Traffic](#27-configure-forwarding-rule-to-receive-traffic)
+   - [2.8 Test Traffic from Link Partner to E2100](#28-test-traffic-from-link-partner-to-e2100)
+
+## 1. Overview
+
+This document explains how to install and run the Infrap4d
+on Intel IPU E2100 hardware.
+
+Note: This document assumes that you have followed
+[Deploying P4 programs guide](https://github.com/ipdk-io/networking-recipe/blob/main/docs/guides/es2k/deploying-p4-programs.md)
+to install the P4 package on FXP during IMC boot-up.
+
+## 2. Running Infrap4d on Arm Compute Complex (ACC)
+
+### 2.1 Extract the tarball to get P4 libraries
+
+Login to ACC and untar the `p4.tar.gz` tarball available under `/opt`
+
+```bash
+[root@mev-acc-rl opt]# cd /opt
+[root@mev-acc-rl opt]# tar -xf p4.tar.gz
+[root@mev-acc-rl opt]# cd /opt/p4/
+[root@mev-acc-rl p4]# ls -lr
+total 8
+drwxr-xr-x 2 root root 4096 Nov 17 09:18 p4sde
+drwxr-xr-x 2 root root 4096 Nov 17 09:18 p4-cp-nws
+```
+
+p4sde and p4-cp-nws binaries and libraries can be found under `/opt/p4/p4sde`
+and `/opt/p4/p4-cp-nws` respectively.
+
+### 2.2 Setup the Environment
+
+Note: `no_proxy` is set to avoid issues with the gRPC server connection.
+The `sudo` option is required when running copy_config_files.sh since
+we are creating directories and copying config files at a system level.
+
+```bash
+export SDE_INSTALL=/opt/p4/p4sde
+export P4CP_INSTALL=/opt/p4/p4-cp-nws
+export DEPEND_INSTALL=$P4CP_INSTALL
+export no_proxy=<system_proxy>,localhost,127.0.0.1,192.168.0.0/16,\
+<Docker container network>/12
+export NO_PROXY=<system_proxy>,localhost,127.0.0.1,192.168.0.0/16,\
+<Docker container network>/12
+unset http_proxy
+unset https_proxy
+
+# Setup the environment
+bash $P4CP_INSTALL/sbin/setup_env.sh $P4CP_INSTALL $SDE_INSTALL $DEPEND_INSTALL
+sudo $P4CP_INSTALL/sbin/copy_config_files.sh $P4CP_INSTALL $SDE_INSTALL
+```
+
+### 2.3 Install TLS Certificates
+
+See [Install TLS Certificates](https://github.com/ipdk-io/networking-recipe/blob/main/docs/guides/install-tls-certificates.md)
+for step by step guide for generating and installing TLS certificates
+
+### 2.4 Generate Forwarding Pipeline Binary
+
+#### 2.4.1 Prepare configuration files
+
+1. Create tofino.bin
+
+```bash
+export OUTPUT_DIR=/opt/p4/p4sde/share/mev_reference_p4_files/simple_l3_l4_pna
+cd $OUTPUT_DIR
+touch tofino.bin
+```
+
+2. Copy `bfrt.json`, `context.json`, `p4info.txt` generated by following
```

Comment:
Minor indentation error from linter.

```
MD029/ol-prefix: Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/2/3]markdownlint[MD029](https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md029.md)
```

Reply:
Documentation: https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md029---ordered-list-item-prefix

MD029 has two modes: one and ordered. The default mode is one which looks for the following style:

1. First
1. Second
1. Third
The benefit of this style is that lines can be reordered without having to worry about the numbers going out of sequence. If you wish to change the style to ordered per your example above, you can update the rule configuration via one of the supported mechanisms:

https://github.com/DavidAnson/markdownlint#configuration
https://github.com/DavidAnson/vscode-markdownlint#configure

So, both are acceptable